src/admin/python/itest_load.py

```diff
@@ -54,22 +54,29 @@ def unzip(source: str, destination: str) -> None:
         shutil.copyfileobj(_source, _target)
 
 
-def execute(cursor: MySQLCursor, stmt: str, multi: bool = False) -> Sequence[Sequence[Union[str, int]]]:
-    """Execute a statement in a cursor, and clean up the cursor
-    so it can be closed."""
+def execute(cursor: MySQLCursor, stmt: str, multi: bool = False):
```

Comment:
would be nice to eliminate repeated code here if we can?

Reply:
done